telegram_bot.py

```diff
@@ -526,6 +527,16 @@ def skip_track(self, update, context):
     def play_pause(self, update, context):
         self.spotify.play_pause()
 
+    def provide_unlistened_songs_details(self, update, context):
+        unlistened_songs_info = ""
+        unlistened_songs = self.personal_messages.get_unlistened_messages()
+        unlistened_songs_info += "You have {} songs waiting to be listened to.\n".format(len(unlistened_songs))
+        count = 1
+        for song in unlistened_songs:
+            unlistened_songs_info += "\nSong {} sent by {} at {}".format(count, song["first_name"], song["timestamp"])
```

Comment:
from now onm use f string pls :)

Reply:
Also can we formate the timestamp a bit, using timstampe.strftime("%Y-%m-%d, %H:%M:%S")?

Reply:
Both good suggestions. Kindly implement and create pr.